test/integration/place_test.rb

```diff
@@ -254,6 +254,29 @@ class PlacesTest < ActionDispatch::IntegrationTest
     end
   end
 
+  context "given an empty postcode" do
+    setup do
+      visit "/passport-interview-office"
+      click_on "Find"
+    end
+
+    should "display error message" do
+      assert page.has_content?("This isn't a valid postcode")
+    end
+
+    should "not show the 'no results' message" do
+      assert page.has_no_content?("We couldn't find any results for this postcode.")
+    end
```

Comment:
Do we need this test? Looking at the logic, only one error can be returned and so testing the presence of that one should be suffice, unless I'm misunderstanding something.

Reply:
The block was copied from the test below, digging into the blames it looks like in the context of the following test it was added to cover regressions that aren't relevant here (as long as they're covered below), so I've removed it.